angular/projects/admin-nrpti/src/app/records/administrative-penalties/administrative-penalty-add-edit/administrative-penalty-add-edit.component.html

```diff
@@ -87,13 +87,17 @@ <h4>Basic Information</h4>
     </section>
 
     <section>
-      <h4>Legislation</h4>
+      <h4>Contravention</h4>
       <app-legislation-add-edit [formGroup]="myForm"></app-legislation-add-edit>
       <div class="flex-container">
         <div class="label-pair">
           <label for="offence">Offence</label>
           <input name="offence" id="offence" type="text" formControlName="offence" class="form-control" />
         </div>
+        <div class="label-pair">
+          <label for="penalty">Penalty</label>
+          <input name="penalty" id="penalty" formControlName="penalty" class="form-control" />
```

Comment:
Should this be a string? Or is it always an amount?

Reply:
I'm investigating this now. If that's the case I will resolve this in a separate PR. It is like this in 4 different models.